### ISSUE TITLE
Text in keybinding widget in high contrast theme is not readable

### ISSUE THREAD

**Issue by ramya-rao-a** [Microsoft/vscode#23504]
Testing #23287

![image](https://cloud.githubusercontent.com/assets/16890566/24428332/f56a1b66-13c2-11e7-9b8d-42b2b280b652.png)
